Project.xml

```diff
@@ -17,7 +17,7 @@
 	<window width="450" height="800" fps="60" background="#000000" hardware="true" vsync="false" />
 
 	<!--HTML5-specific-->
-	<window if="html5" resizable="false" />
+	<window if="html5" resizable="true" />
```

Comment:
@jacque006 @thavelick This was why it didn't render correctly on web fullscreen....

Reply:
Do we want to follow up by adding those swipe/tap controls? Or does it already work?

Reply:
Heh, that's easy. Swipe controls do indeed work as far as I could tell.

Reply:
Noice!

Reply:
I put swipe controls in this morning, heh. It was super simple (might not work for other games... but worked for this one)